P0 screens

rdshinde opened the pull request:
Added navbar for landing page.
Also added root color variables.

<img width="1434" alt="Screenshot 2022-02-14 at 9 56 17 PM" src="https://user-images.githubusercontent.com/67017632/153904431-9c87148e-fc36-4ee9-8345-04e047ca792c.png">

rdshinde committed:
initial commit

rdshinde committed:
feat: navbar added

rdshinde committed:
Update style.css

ttahm3d commented on index.html:
```diff
@@ -0,0 +1,103 @@
+            <div class="nav__logo-container">
+              <a href="./index.html" class="text-default"
+                ><i class="fas fa-store text-1"></i>&nbsp;
+                <h1 class="">
+                  Swadeshi
+                  <h4 class="bold-lg">.com</h4>
+                </h1>
```
Dont use one heading tag inside another tag, Use span to do these kind of things

ttahm3d commented on index.html:
```diff
@@ -0,0 +1,103 @@
+            </div>
+            <ul class="nav__links m-x-md">
+              <li class="m-x-sm p-x-sm">
+                <a class="btn btn-link p-b-lg" href="">Men</a>
```
Dont leave empty `gref` in `a` tag. Use `#` if you dont have any thing to reference to.

ttahm3d reviewed (commented):
Left few comments please take a look.

theshivamverma commented on style.css:
```diff
@@ -0,0 +1,90 @@
+}
+
+/* Stling for nav__logo text */
+a h1,
+a h4 {
+  display: inline-block;
+}
+a .fa-store:hover {
+  transform: scale(1);
+}
```
Use CSS styles to format the UI, instead of using random tags together.

rdshinde replied:
Refactored this code and fixed all semantic error

theshivamverma commented on index.html:
```diff
@@ -0,0 +1,103 @@
+      </div>
+    </div>
+
+    <script
+      src="https://kit.fontawesome.com/fde0352739.js"
+      crossorigin="anonymous"
+    ></script>
```
Use cdn in head to load external libraries.

rdshinde replied:
Added CDN to head tag.

theshivamverma commented on style.css:
```diff
@@ -0,0 +1,90 @@
+.header-container {
+  position: sticky;
+  top: 0;
+  z-index: 999;
```
can store z-indexes in variables and use, avoid random large numbers.

rdshinde replied:
Created Variables for z-index.

rdshinde committed:
Refactored: Made changes which were told in review.

- Added Fontawesome CDN in head-tag
- Fixed Semantic errors
- Created variables for z-index
